os/drivers/audio/alc5658.c

```diff
@@ -674,8 +674,8 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct
 	}
 
 	alc5658_setregs(priv);
-
 	alc5658_getregs(priv);
```

Comment:
why do you setresgs and then getregs ?
Is it the code that checks to see if it is set?

Reply:
alc5658_getregs can be used to get the register values anytime for debugging alc5658.